From ce49a517622d2babc01d7430ac0a1784338629c0 Mon Sep 17 00:00:00 2001
From: Huy Hoang <boydaihungst@gmail.com>
Date: Wed, 1 Nov 2023 22:20:21 +0700
Subject: [PATCH] Remove duplicated system props

---
 base.mk     | 1 -
 system.prop | 3 ---
 2 files changed, 4 deletions(-)

diff --git a/base.mk b/base.mk
index 2c4f927..cf45a3c 100644
--- a/base.mk
+++ b/base.mk
@@ -37,7 +37,6 @@ PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
 	ro.build.version.all_codenames=$(PLATFORM_VERSION_ALL_CODENAMES) \
 	ro.build.version.release=$(PLATFORM_VERSION) \
 	ro.build.version.security_patch=$(PLATFORM_SECURITY_PATCH) \
-	ro.adb.secure=0 \
 	ro.logd.auditd=true \
 	ro.logd.kernel=true \
 
diff --git a/system.prop b/system.prop
index f3c0c45..968a412 100644
--- a/system.prop
+++ b/system.prop
@@ -1,4 +1,3 @@
-ro.adb.secure=0
 #Some devices have sdcardfs kernel panicing on 8.0
 #Disable for everyone for the moment
 ro.sys.sdcardfs=0
@@ -8,8 +7,6 @@ persist.sys.overlay.nightmode=true
 persist.sys.strictmode.disable=true
 persist.sys.phh.mainkeys=0
 
-fw.max_users=10
-persist.sys.max_profiles=10
 ro.boot.realme.lockstate=0
 
 #Fixes fingerprint unlock delay
-- 
2.42.0

