From e6e0910ef6269aedad982e0de9af54befb35e4cb Mon Sep 17 00:00:00 2001
From: Huy Hoang <boydaihungst@gmail.com>
Date: Tue, 31 Oct 2023 22:25:26 +0700
Subject: [PATCH] mini gms

---
 gms_mini.mk | 15 +--------------
 1 file changed, 1 insertion(+), 14 deletions(-)

diff --git a/gms_mini.mk b/gms_mini.mk
index 9f991d3..19168f3 100644
--- a/gms_mini.mk
+++ b/gms_mini.mk
@@ -24,12 +24,7 @@ endif
 
 # product/app
 PRODUCT_PACKAGES += \
-    CalculatorGooglePrebuilt \
-    CalendarGooglePrebuilt \
-    Chrome-Stub \
-    Chrome \
     GoogleContacts \
-    GoogleTTS \
     LatinIMEGooglePrebuilt \
     LocationHistoryPrebuilt \
     MarkupGoogle \
@@ -40,25 +35,19 @@ PRODUCT_PACKAGES += \
     SoundPickerPrebuilt \
     TrichromeLibrary-Stub \
     TrichromeLibrary \
-    WallpaperEmojiPrebuilt \
     WebViewGoogle-Stub \
-    WebViewGoogle \
-    arcore \
-    talkback
+    WebViewGoogle
 
 # product/priv-app
 PRODUCT_PACKAGES += \
     AICorePrebuilt \
-    AiWallpapers \
     AdaptiveVPNPrebuilt \
-    AndroidAutoStubPrebuilt \
     AppDirectedSMSService \
     BetterBugStub \
     CarrierLocation \
     ConfigUpdater \
     DeviceIntelligenceNetworkPrebuilt \
     DevicePersonalizationPrebuiltPixel2020 \
-    FilesPrebuilt \
     GCS \
     GoogleDialer \
     GoogleOneTimeInitializer \
@@ -71,7 +60,6 @@ PRODUCT_PACKAGES += \
     PartnerSetupPrebuilt \
     Phonesky \
     PrebuiltBugle \
-    SafetyHubPrebuilt \
     SettingsIntelligenceGooglePrebuilt \
     SetupWizardPrebuilt \
     TurboPrebuilt \
@@ -85,7 +73,6 @@ PRODUCT_PACKAGES += \
 
 # system/priv-app
 PRODUCT_PACKAGES += \
-    DocumentsUIGoogle \
     GooglePackageInstaller \
     TagGoogle
 
-- 
2.42.0

